### ISSUE TITLE
Experiencing similar issues to #7

### ISSUE THREAD

**Issue by cole007** [mikestecker/craft3-videoembedder#22]
We are using this plugin and recently we have started having the plugin throw an Internal Server Error when embedding Vimeo videos.

The logs are showing `Embed\Exceptions\InvalidUrlException: Invalid url 'https://vimeo.com/294536324' (Status code 403)` but unlike Issue #7 this video has no restrictions on access or embedding. 

Not at all sure why this particular video would be throwing a 403 error from the Embed service.

Any thoughts on resolution appreciated. 

Craft CMS 3.3.0.1
Video Embedder 1.1.3
PHP version 7.1.31

**Comment by cole007** [comment]
For further info this may be an issue with the Embed library - see issue at https://github.com/oscarotero/Embed/issues/326

**Comment by b3nhartl3y** [comment]
Yeah, we are getting this as well. Sad times. Are you on Digital Ocean @cole007?

**Comment by cole007** [comment]
@b3nhartl3y yep - have had to fallback on a simple macro for time being:

```
{%- macro video(url) -%}
    {% if 'youtu' in url %}
        {% set id = '?v=' in url ? url\|split('?v=') : url\|split('youtu.be/') %}
        {% set embedCode = id is iterable ? 'https://www.youtube.com/embed/' ~ id[1] : null %}
    {% elseif 'vimeo' in url %}
        {% set id = url\|split('vimeo.com/') %}
        {% set embedCode = id is iterable ? 'http://player.vimeo.com/video/' ~ id[1] : null %}
    {% endif %}
    {% if embedCode %}
       {{ ('<iframe src="' ~ embedCode ~ '"></iframe>')|raw }}
    {% endif %}
{%- endmacro -%}
```

Would be simple enough to pass additional parameters into the macro if needed as per the plugin.

**Comment by engram-design** [comment]
Sorry to say that while we've taken over this plugin with the intention to bring it to Craft 4+, we're not going to be continuing development on this plugin. In addition, we're no longer supporting Craft 3 plugins due to Craft itself reaching its EOL.

But the good news is we've got an even better plugin (we think) for Craft 4 and Craft 5 called [Video Picker](https://verbb.io/blog/video-picker) which has the same [embeddable videos](https://verbb.io/craft-plugins/video-picker/docs/template-guides/embedding-videos) feature.

The even better news is that we're happy to gift anyone a **free** license for any site they have with Video Embedder installed, to ease the migration to Craft 4/5. Just [get in touch](https://verbb.io/contact) with a screenshot of your current site with Video Embedder installed as proof of upgrade.

This repository will still exist as a reference, PR's can still be made, and releases completed, but there won't be any active development from us.
